Bots/AuthServer.js

```diff
@@ -62,22 +55,22 @@ function AuthServer(){
   this.app.use(session({
     secret: sessinCredential,
     cookie: {
-      maxAge: 30 * 24 * 60 * 60 * 1000
-    }
+      maxAge: 30 * 24 * 60 * 60 * 1000,
+    },
   }));
 
   this.app.use(passport.initialize());
   this.app.use(passport.session());
 
-  router.get('/twitter', passport.authenticate('twitter'), function(req, res){
+  router.get('/twitter', passport.authenticate('twitter'), (req, res)=> {
     res.send('twitter-ok');
   });
 
-  router.get('/twitter/callback', passport.authenticate('twitter', { failureRedirect: '/fail' }), function(req, res){
+  router.get('/twitter/callback', passport.authenticate('twitter', { failureRedirect: '/fail' }), (req, res)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).